numpy/lib/function_base.py

```diff
@@ -1491,20 +1491,30 @@ def gradient(f, *varargs, **kwargs):
     """
     Return the gradient of an N-dimensional array.
 
-    The gradient is computed using second order accurate central differences
-    in the interior and either first differences or second order accurate
-    one-sides (forward or backwards) differences at the boundaries. The
-    returned gradient hence has the same shape as the input array.
+    The gradient (defined as :math:`\\nabla f = [\\frac{\\partial f}{\\partial x},  
```

Comment:
You could do something like "The gradient of a function is the vector formed by its partial derivatives with respect to each of its independent variables."